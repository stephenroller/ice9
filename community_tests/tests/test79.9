# proc () ; end ; (missing id)								

proc 
(
) 
; 
end 
;
