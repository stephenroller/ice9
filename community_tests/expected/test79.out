Loading file: generated/test79.tm
Number of instructions executed = 4
